### PR TITLE
Minimize to tray

### PR DESCRIPTION
**Pull request by marc0tjevp**

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -25,7 +26,8 @@ async function createWindow() {
 
 	win = new BrowserWindow({
 		title: 'LISTEN.MOE - Desktop App',
-		icon: 'public/logo.png',
+		// eslint-disable-next-line no-undef
+		icon: join(__static, 'logo.png'),
```
Comment (marc0tjevp) [inline src/background.js]: This ensures the application logo is loaded for other windows too, it's a weird one.

File: vue.config.js [hunk 2]
```diff
@@ -15,7 +15,8 @@ module.exports = {
 								'x64'
 							]
 						}
-					]
+					],
+					icon: 'build/icon.ico'
```
Comment (iCrawl) [inline vue.config.js]: Isn't specifying this redundant? Also the subsequent calls?

File: src/background.js [hunk 3]
```diff
@@ -84,7 +101,9 @@ async function createWindow() {
 
 		loginModal = new BrowserWindow({
 			width: 350,
-			height: 500,
+			height: 400,
+			// eslint-disable-next-line no-undef
+			icon: join(__static, 'logo.png'),
```
Comment (marc0tjevp) [inline src/background.js]: Same here.

File: vue.config.js [hunk 4]
```diff
@@ -25,7 +26,8 @@ module.exports = {
 								'x64'
 							]
 						}
-					]
+					],
+					icon: 'build/256x256.png'
```
Comment (iCrawl) [inline vue.config.js]: Apart from this one, macOS actually wants a 512x512 lmao

File: src/background.js [hunk 5]
```diff
@@ -110,7 +129,9 @@ async function createWindow() {
 
 		settingsModal = new BrowserWindow({
 			width: 500,
-			height: 595,
+			height: 760,
+			// eslint-disable-next-line no-undef
+			icon: join(__static, 'logo.png'),
```
Comment (marc0tjevp) [inline src/background.js]: Same here.

File: src/components/settings/index.vue [hunk 6]
```diff
@@ -51,6 +65,12 @@
 					@clicked="setOption('eventStarts')">Desktop notification when an event starts</Toggle>
 				<Toggle :active="alwaysOnTop"
 					@clicked="setOption('alwaysOnTop')">Always on top</Toggle>
+				
+				<h1 class="title">Tray settings</h1>
+				<Toggle :active="hideFromTaskbar"
```
Comment (marc0tjevp) [inline src/components/settings/index.vue]: This toggle doesn't do it's job right now, it's the one thing that's not finsihed

File: vue.config.js (hunk 2 above)
Comment (marc0tjevp) [reply vue.config.js]: It's because of the extension. Because Linux wants a png I moved the ico to Windows only.

File: vue.config.js (hunk 4 above)
Comment (marc0tjevp) [reply vue.config.js]: Seems like the merge from upstream changed it back, I'll change it back this weekend when I try to finish this PR :eyes:

**Comment by Pitu** [comment]
See #21

**Pull request closed without merging**